scripts/check-peer-constraints.go

```diff
@@ -36,6 +36,7 @@ var Peers = []string{
 	"storj.io/storj/bootstrap/...",
 	"storj.io/storj/versioncontrol/...",
 	"storj.io/storj/linksharing/...",
+	"storj.io/storj/pkg/certificates/...",
```

Comment:
Note, peers are kept at the top-level. So `storj.io/storj/certificate`.

The package name is `certificate`, I suspect this is a reason why it's not showing errors.

Reply:
Are you saying we'd have to move storj/pkg/certificates to storj/certificates first?

Reply:
Yes, because `pkg/certificate` is part of `pkg/...` which is considered a library. This would mean that a library has a dependency on the peer.

Reply:
Gotcha. That makes sense.